dojson/utils.py

```diff
@@ -13,6 +13,25 @@
 import six
 
 
+class IgnoreKey(Exception):
+
+    """The corresponding key has been ignored."""
+
+    pass
+
+
+def ignore_value(f):
```

Comment:
@jma yes, just few cosmetics. Can you please add `.. versionadded:: 0.2.0` (see http://sphinx-doc.org/markup/para.html#directive-versionadded)